index.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Quiz Works</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+		
+		<article class="salescard">
+			<header class="scheader">
+				<h1>Spanky Cheese</h1>
+			</header>
+
+				<section class="section1">
+					<div id="pic">
+						<img class="displayed" src="https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcSe3GXnP1NAiFVYPkqhhTLvkf_uPfrbkbI0GwDsqtvA0W_tKldfsg" height="60" width="60">
+					</div>
+			
+					<div class="description">
+					Bacon ipsum dolor amet capicola short loin beef t-bone jowl chuck. Meatloaf sirloin shank meatball spare ribs bacon. Kielbasa chuck spare ribs, drumstick tongue shoulder bresaola jerky porchetta rump short ribs doner biltong. Hamburger beef boudin ground round venison porchetta shankle andouille, drumstick turducken turkey sausage spare ribs strip steak corned beef.  Meatball ribeye meatloaf jerky venison porchetta frankfurter boudin.
+				
+					</div>
+
+
+					<div class="availability">
+					available
+				
+					</div>
+				</section>
+				
+
+				<section class="specs">
+						<div>
+					    <h1 class="spechead"> Specifications </h1>
```

Comment:
You are mixing tabs and spaces. Which do you prefer? Keep consistent. We will discuss in class later.